### PR TITLE
Refactoring the code

### PR DESCRIPTION
**Pull request by danielaSantaCruz**
Encapsulation of functions

### CODE REVIEW

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java [hunk 1]
```diff
@@ -27,6 +27,75 @@ public CoffeeMaker() {
...
 
+    /**
+     * Fill the boiler with cups of water
+     */
+    public void fillBoilerWithWater(final int cupsWater) {
+        boiler.setCupsOfWater(cupsWater);
+    }
+
+    /**
+     * Fill the boiler with cups of water
```
Comment (cristianfchq) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: Fill the boiler with cups of water ??????????
This is not a good reference for the method

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java [hunk 2]
```diff
@@ -27,6 +27,75 @@ public CoffeeMaker() {
...
+    public boolean obtainsStartButtonState() {
+        return startButton.getButtonPressed();
+    }
+
+    /**
+     * Returns true if the pot is it's place
+     * @return boolean
+     */
+    public void pressStartButton() {
+        startButton.isPressed();
```
Comment (edsonarios) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: The name of this method seems to ask but it is not a boolean

Comment (danielaSantaCruz) [reply src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: The name of the method asserts the action of pressing the button, is not assking if the button is pressed

Comment (edsonarios) [reply src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: the rest of the Boolean type methods start in the same way, but in this case it is not Boolean so it does not seem consistent

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java (hunk 1 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: Thanks for the observation

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java [hunk 3]
```diff
@@ -27,6 +27,75 @@ public CoffeeMaker() {
...
+    /**
+     * Returns true if the pot is it's place
+     * @return boolean
+     */
+    public boolean obtainsStartButtonState() {
+        return startButton.getButtonPressed();
+    }
+
+    /**
+     * Returns true if the pot is it's place
```
Comment (edsonarios) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: This documentation seems to be wrong

Comment (danielaSantaCruz) [reply src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: Thanks for the observation, I already corrected it.

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java [hunk 4]
```diff
@@ -27,6 +27,75 @@ public CoffeeMaker() {
...
+     * @return boolean
+     */
+    public boolean isPotInPlace() {
+        return pot.getIsInPlace();
+    }
+
+    /**
+     * Sets the new place of the pot
```
Comment (cristianfchq) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: could you explain me why the coments and the method. I dont understand. pot in new place???? What is the new place of pot?????

I thing that the name method can be better removePot

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java (hunk 2 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: Ok I'll review the method name and change it

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java (hunk 4 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/CoffeeMaker.java]: Ok i'll explain better that method

**Review by cristianfchq — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]
Good job

**Pull request merged**